src/App.jsx

```diff
@@ -19,10 +26,15 @@ export function App() {
 	 * to create and join a new list.
 	 */
 	const [listToken, setListToken] = useStateWithStorage(
-		'my test list',
+		null,
 		'tcl-shopping-list-token',
 	);
 
+	const handleClick = useCallback(() => {
```

Comment:
I think this useCallback is on the edge of premature optimization, but, still, it's cool to see you thinking about maintaining stable references to event handler functions.